editor/plugins/script_editor_plugin.cpp

```diff
@@ -1141,14 +1138,13 @@ void ScriptEditor::_menu_option(int p_option) {
 		} break;
 		case TOGGLE_SCRIPTS_PANEL: {
 			if (current) {
-				CodeTextEditor *code_editor = NULL;
 				ScriptTextEditor *editor = dynamic_cast<ScriptTextEditor *>(current);
```

Comment:
Does this need to handle `TextEditor` and `VisualScripts`? 

To keep constancy should use ` Object::cast_to<ScriptTextEditor>(current)`.

Reply:
> Does this need to handle TextEditor and VisualScripts?

Exactly.

> To keep constancy should use Object::cast_to<ScriptTextEditor>(current).

Fixed.